src/tasks/minions/alchingActivity.ts

```diff
@@ -0,0 +1,69 @@
+import { AlchingActivityTaskOptions } from '../../lib/types/minions';
+import { KlasaMessage, Task } from 'klasa';
+import { channelIsSendable } from '../../lib/util/channelIsSendable';
+import { resolveNameBank, toKMB } from 'oldschooljs/dist/util';
+import { noOp, roll, saidYes } from '../../lib/util';
+import getUsersPerkTier from '../../lib/util/getUsersPerkTier';
+import { Time } from '../../lib/constants';
+import hasItemEquipped from '../../lib/gear/functions/hasItemEquipped';
+import { UserSettings } from '../../lib/settings/types/UserSettings';
+import getOSItem from '../../lib/util/getOSItem';
+
+const { id: bryophytasStaffId } = getOSItem("Bryophyta's staff");
+
+export default class extends Task {
+	async run({ itemName, quantity, channelID, alchValue, userID }: AlchingActivityTaskOptions) {
+		let savedRunes = 0;
```

Comment:
move the declaration of `savedRunes` below above the if statement

Reply:
Moving this below the if breaks things because it's required by the responses